prowler/providers/aws/lib/session/aws_set_up_session.py

```diff
@@ -192,10 +192,8 @@ def validate_arguments(
                 "If a role ARN is provided, a session duration, an external ID, and a role session name are required."
             )
     else:
-        if session_duration or external_id or role_session_name:
-            raise ValueError(
-                "If a session duration, an external ID, or a role session name is provided, a role ARN is required."
-            )
+        if external_id:
```

Comment:
Why this change?

Reply:
Because now the role session name and the session duration will have always a value.